examples/connections/azure_ai_language.yml

```diff
@@ -0,0 +1,7 @@
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/CustomConnection.schema.json
+name: azure_ai_language_connection
```

Comment:
Could you please also put these two connection files into `flows/integrations/azure-ai-languange/connections` folder? They are not officially supported connections and should not be in shared connection folder.